source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Evaluations/Calification.cshtml

```diff
@@ -150,7 +150,10 @@
     </tbody>
   </table>
   <div class="footer-evaluation" data-bind="if: isEvaluationEditable()">
-    <button class="button-evaluations primary-btn finish-btn" data-bind="click: onFinish">Finalizar</button>
+    <button class="button-evaluations primary-btn finish-btn" data-bind="click: onFinish">
+      <span data-bind="if: isResposibleCalificating()"> finalizar e iniciar evaluación empresa </span>
+      <span data-bind="if: !isResposibleCalificating()"> finalizar</span>
```

Comment:
Yes, i can remove the spams and use the comment tags. Space before "Finalizar" is not needed neither